package.xml

```diff
@@ -1,7 +1,7 @@
 <package>
 
   <name>axis_camera</name>
-  <version>0.4.2</version>
+  <version>0.4.3</version>
```

Comment:
Just revert this, while blooming it will increase the version automatically.

Reply:
Reverted